sormas-e2e-tests/src/test/java/org/sormas/e2etests/entities/services/SampleService.java

```diff
@@ -166,4 +166,16 @@ public Sample buildPathogenTestUnverifiedDE(String testType) {
         .testResultsComment("Comment on Edit Pathogen requests or received " + currentTimeMillis)
         .build();
   }
+
+  public Sample buildGeneratedPositiveSampleDE() {
+    return Sample.builder()
+        .purposeOfTheSample("INTERNER /IN-HOUSE TEST")
+        .dateOfCollection(LocalDate.now().minusDays(10))
+        .laboratory("Labor")
+        .testedDisease("COVID-19")
+        .sampleTestResults("Positiv")
+        .resultVerifiedByLabSupervisor("JA")
+        .sampleType("Nasen-Abstrich")
+        .build();
```

Comment:
let's make this method reusable. Use random data for those fields where we can do this, and parametrize the method to get the test result value from there. If COVID is needed, add also this value as parameter

Reply:
There isn't really any way to make this reusable, aside from parametrizing every possible field, and then putting every value you're seeing above into the test step as parameters (which would make the step definition ridiculously long).

- In-house test and Labor are required to prevent additional required fields from appearing
- COVID-19 is the only disease available in test-de
- Nasal swab, positive verification by supervisor and positive test result are required by the test scenario to produce expected results in the case view

 If it were possible to vary some of that data, rest assured that I would have done so :-). This scenario however requires some very specific data across multiple fields, so it's not practically viable to randomize or parametrize it.